inst/shiny/modules/tab_nca/nca_setup.R

```diff
@@ -28,43 +61,35 @@ nca_setup_ui <- function(id) {
           ),
           # Method, NCA parameters, and units table
           fluidRow(
-            column(3, selectInput(
+            column(4, selectInput(
               ns("method"),
               "Extrapolation Method:",
               choices = c(
                 "lin-log", "lin up/log down", "linear"
               ),
               selected = "lin up/log down"
             )),
-            column(5, pickerInput(
-              inputId = ns("nca_params"),
-              label = "NCA parameters to calculate:",
-              choices = pull(pknca_cdisc_terms, PKNCA, input_names),
-              options = list(
-                `live-search` = TRUE,
-                `dropup-auto` = FALSE,
-                `size` = 10,
-                `noneSelectedText` = "No parameters selected",
-                `windowPadding` = 10,
-                `dropdownAlignRight` = TRUE
-              ),
-              multiple = TRUE,
-              selected = c("cmax", "tmax", "half.life", "cl.obs", "auclast",
-                           "aucinf.pred", "aucinf.obs", "aucinf.obs.dn",
-                           "adj.r.squared", "lambda.z", "lambda.z.n.points",
-                           "cav", "cl.all", "cl.obs",
-                           "clast", "tlast")
-            )),
+            column(4, #pickerinput only enabled when IV and EX data present
+                   shinyjs::hidden(
+                     pickerInput(
+                       ns("bioavailability"),
+                       "Calculate Bioavailability:",
+                       choices = c("f_aucinf.obs", "f_aucinf.pred", "f_auclast"),
+                       multiple = TRUE,
+                       selected = NULL
+                     )
+                   )),
             column(4, units_table_ui(ns("units_table")))
           ),
-          #pickerinput only enabled when IV and EX data present
-          shinyjs::hidden(
-            pickerInput(
-              ns("bioavailability"),
-              "Calculate Bioavailability:",
-              choices = c("f_aucinf.obs", "f_aucinf.pred", "f_auclast"),
-              multiple = TRUE,
-              selected = NULL
+        ),
+        accordion_panel(
+          title = "Parameter Selection",
+          reactableOutput(ns("nca_parameters")),
+          card(
+            full_screen = FALSE,
+            card_header("Selected NCA Parameters"),
```

Comment:
nitpick: For me, if all options in the table are open, the table seems a little bit too close to the card with pills - I would consider adding a small margin for the spacing to look a bit nicer:
```suggestion
            full_screen = FALSE,
            style = "margin-top: 2em;",
            card_header("Selected NCA Parameters"),
```